### PR TITLE
TMSFPW-9: add landing page

### PR DESCRIPTION
**Pull request by noexanu**

### CODE REVIEW

File: src/components/landingPage/LandingPage.styles.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const LandingPageContentContainer = styled.div`
+  margin-top: 75px;
```
Comment (paistitshare) [inline src/components/landingPage/LandingPage.styles.ts:4]: It's better to define paddings and margins within single css property definition like so:
`margin: 75px 0 25px;`.
margin-%sidename% definitions are also fine if they are used once per selector.

File: src/components/landingPage/contentSection/ContentSection.styles.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  max-width: 800px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 20px;
+  padding: 0 100px;
+  font-family: Arial, Helvetica, sans-serif;
+`;
+
+export const LandingPageContentSectionHeadr = styled.h2`
```
Comment (paistitshare) [inline src/components/landingPage/contentSection/ContentSection.styles.ts:29]: LandingPageContentSectionHead**e**r

File: src/components/landingPage/navbar/Navbar.styles.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  overflow: hidden;
+  box-shadow: 0 0 5px rgba(0, 0, 0, .25);
+  background-color: #fff;
+`;
+
+export const LandingPageNavbarDropdownMenuButton = styled(LandingPageNavbarButton)`
+  width: 100%;
+  height: 40px;
+  text-align: left;
+  padding-left: 20px;
```
Comment (paistitshare) [inline src/components/landingPage/navbar/Navbar.styles.ts:75]: same

File: src/components/landingPage/navbar/Navbar.styles.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  height: fit-content;
+`;
+
+export const LandingPageNavbarUserIconContainer = styled.div`
+  height: 40px;
+  aspect-ratio: 1 / 1;
+  border-radius: 50%;
+  overflow: hidden;
+  background-color: #829ec2;
+  & * {
```
Comment (paistitshare) [inline src/components/landingPage/navbar/Navbar.styles.ts:50]: can this selector be more specific?

**Review by paistitshare — CHANGES_REQUESTED** [review]
Let's use "Home Page" term instead of "Landing Page"
Here's some related info for a reference: https://landingi.com/pl/tworzenie/landing-page/

File: src/components/landingPage/contentSection/ContentSection.styles.ts (hunk 2 above)
Comment (noexanu) [reply src/components/landingPage/contentSection/ContentSection.styles.ts:29]: Fixed

File: src/components/landingPage/navbar/Navbar.styles.ts (hunk 3 above)
Comment (noexanu) [reply src/components/landingPage/navbar/Navbar.styles.ts:75]: Fixed

File: src/components/landingPage/LandingPage.styles.ts (hunk 1 above)
Comment (noexanu) [reply src/components/landingPage/LandingPage.styles.ts:4]: Fixed

File: src/components/landingPage/navbar/Navbar.styles.ts (hunk 4 above)
Comment (noexanu) [reply src/components/landingPage/navbar/Navbar.styles.ts:50]: fixed

**Review by paistitshare — APPROVED** [review]

**Pull request merged**